### ISSUE TITLE
关于DST equation 7 （Debiased Self-Training for Semi-Supervised Learning）

### ISSUE THREAD

**Issue by gufan-d** [thuml/Transfer-Learning-Library#220]
作者您好，抱歉打扰了。我刚接触半监督学习，您的DST工作对我启发很大，有关于公式7的实现有些不明白想请教您一下。
```
    def forward(self, y_l, y_l_adv, y_u, y_u_adv):
        _, prediction_l = y_l.max(dim=1)
        loss_l = self.eta_prime * F.cross_entropy(y_l_adv, prediction_l)

        _, prediction_u = y_u.max(dim=1)
        loss_u = F.nll_loss(shift_log(1. - F.softmax(y_u_adv, dim=1)), prediction_u)

        return loss_l + loss_u
```
这里1. - F.softmax(y_u_adv, dim=1)是要鼓励loss_u向相反的方向收敛吗？不太明白
希望能得到您的指点，不胜感激！
